alexandra_nechita/7th.2.Python_Homework.py

```diff
@@ -0,0 +1,15 @@
+def safe_divide(divide):
+    def safe_divide_inner(first_number, second_number):
+        if second_number == 0:
+            print("Division cannot be performed")
+            return
+
+        return print(divide(first_number, second_number))

```

Comment:
daca folosim print functia o sa returneze un string in loc de `int` o functie care face o impartire ne asteptam tot timpul sa returneze un `int` sau un `float`. Aici poti sa returnezi direct `divide(first_number, second_number)`